executor/log.go

```diff
@@ -0,0 +1,48 @@
+package executor
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/screwdriver-cd/sd-cmd/config"
+	"github.com/screwdriver-cd/sd-cmd/util"
+)
+
+var logFile *os.File
+
+// StartLog make the log output to the file and stderr
```

Comment:
Please teach me about this behavior, even if sd_cmd command is succeeded, it output only stderr and log file?

Reply:
Sorry this function name is not good. Yes even if sd_cmd command is succeeded, sd-cmd output both stderr and log file.

This function prepare for executor logging. 

First I create a log file in this way `$SD_ARTIFACTS_DIR/.sd/commands/namespace/name/version/timestamp.log`.

Then make golang library `log package` output log to stderr and the log file.

So after this was called when we use log.Print* they output stderr and the log file.

Reply:
Is it better to change the name of this function?